tests/integration/server/api/users_api_test.py

```diff
@@ -155,19 +156,24 @@ def test_update_manager():
         user2_response = create_test_user(session, user2_payload)
         user2_result = assert_api_success(user2_response)
         user2_id = user2_result["data"]["user"]["id"]
+
+        next_admins = {
+            "name": "NextAdmins",
+            "owners": [user2_id],
+            "administrators": [user2_id],
+        }
+        role_response = create_test_role(session, next_admins)
+        add_role_member(session, role_response.json()["data"]["id"], {"id": user2_id})
+
         manager_payload = {
-            "id": user2_id,
-            "reason": "Integration test of adding role owner.",
+            "id": user1_id,
+            "reason": "Integration test of updating manager.",
             "metadata": "",
         }
-        response = session.put(
-            "http://rbac-server:8000/api/users/{}/manager".format(user1_id),
-            json=manager_payload,
-        )
+        response = update_manager(session, user2_id, manager_payload)
```

Comment:
Can you also include a bad test case where a user that is attempting to hit the update manager's API endpoint is not a Next Admin? To make sure your functionality is preventing non-Next admins from making the update Manager proposals? 

You can just copy line 173 to before line 166. This way, you try to hit the update manager API endpoint before user2 is in the NextAdmins role.